content/F2016/F16-event-gooutside.md

```diff
@@ -0,0 +1,20 @@
+title: WiCS and CSC Go Outside
+Date: 2016-10-13 18:00
+Category: Events
+Tags: social
+Slug: go-outside-s16 
+Author: Shivangi Patwardhan 
+Summary: We're having a bonfire with the CSC. S'mores and a bonfire!
+
+WiCS and CSC are super excited for this upcoming event! We hope that 
+you all can make it and bring your friends!
+
+There will be free s'mores and light dinner will be served :)
+
+## Event Details ##
+
++ **Who:** Everyone
++ **What:** Bonfire!
++ **Where:** Columbia Lake 2 Firepit
++ **When:** Thursday October 13 6:00
++ **Facebook Event Page:** Click [here](https://www.facebook.com/events/534872003376714/).
```

Comment:
the period looks a bit weird on https://wics-site-pr-267.herokuapp.com/ cause no other lines end in a period

and you can still make this line < 80 chars but putting `[here](https://www.facebook.com/events/534872003376714/)` on the next line :)